modules/basegl/CMakeLists.txt

```diff
@@ -85,6 +90,20 @@ set(HEADER_FILES
     include/modules/basegl/properties/linesettingsproperty.h
     include/modules/basegl/properties/splitterproperty.h
     include/modules/basegl/properties/stipplingproperty.h
+    include/modules/basegl/raycasting/atlascomponent.h
+    include/modules/basegl/raycasting/backgroundcomponent.h
+    include/modules/basegl/raycasting/cameracomponent.h
```

Comment:
The only use case so far is for the raycaster, but I think it might have other uses too. But I think most processors should not use this.
Renamed RaycasterComponent to ShaderComponents and use a shadercomponents folder instead

Reply:
Will also rename VolumeRaycasterBase to ShaderComponentProcessorBase